.github/workflows/ci.yml

```diff
@@ -8,46 +8,45 @@ on:
 
 defaults:
   run:
-    shell: bash # https://github.com/beeware/briefcase/pull/912
+    shell: bash  # https://github.com/beeware/briefcase/pull/912
 
 jobs:
   verify-apps:
     name: Build apps
+    runs-on: windows-latest
     strategy:
       fail-fast: false
       matrix:
-        python_version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
While we're here, we should probably add 3.12-dev

Reply:
I don't think this'll work unless a binary stub is also created for 3.12.

Reply:
Ah - good point; and we can't do that until there's a NuGET release of Python 3.12.